client/src/components/Register.js

```diff
@@ -1,9 +1,22 @@
 import React from 'react';
 import { AuthConsumer, } from "../providers/AuthProvider";
-import { Button, Form, Segment, Header, } from 'semantic-ui-react';
+import { Button, Form, Segment, Header, Radio, } from 'semantic-ui-react';
 
 class Register extends React.Component {
   state = { email: '', name: '', role: '', password: '', passwordConfirmation: '', };
+  
+ friendOptions = () => [
```

Comment:
Maybe change this from friendOptions to something more fitting.

Reply:
On it! Thanks.